### DISCUSSION TITLE
Rust Prisma Client

### DISCUSSION THREAD

**Discussion by daniloraisi** [prisma/prisma#2866] [Q&A] [answered]
I tried to find some open issues related to this, but no success!

Any plans to provide a Prisma client for Rust applications?

**Comment by ryands17** [comment]
Hey @daniloraisi :wave: 
There is no official rust client, but someone from the community has made one here: https://github.com/seunlanlege/prisma-client-rs

**Reply by jruiseco** [reply]
With Rust really starting to hit it's stride. As an avid Prisma user, I would be overjoyed to see an official implementation.

I would also think that with the engine itself written in rust, you might be able to implement the Prisma client in rust via ABI. Which would make it somewhere beyond insanely fast.

**Reply by pantharshit00** [reply]
Hey @jruiseco 

For now, you can checkout our SQL abstraction for Rust which is also opensource: https://github.com/prisma/quaint

**Comment by longfellowone** [comment]
Add another interested name to the list!

**Comment by renhiyama** [comment]
Another one

**Comment by janpio** [comment]
Please use the feature request issue to express your support, we are not monitoring GitHub discussions for that: https://github.com/prisma/prisma/issues/5790

**Comment by jharrell** [comment, marked as answer]
Gonna go ahead and close this in favor of #5790 

Thank you very much for the discussion everyone! Please 👍 that issue to show your support.
